src/Workspaces/Core/Portable/Options/SerializableOptionSet.cs

```diff
@@ -253,6 +252,10 @@ public void Serialize(ObjectWriter writer, CancellationToken cancellationToken)
                     RoslynDebug.Assert(value != null);
                     writer.WriteInt32((int)value);
                 }
+                else if (kind == OptionValueKind.CodeStyleOption || kind == OptionValueKind.NamingStylePreferences)
+                {
+                    ((IObjectWritable)value!).WriteTo(writer);
```

Comment:
Code in the if part uses `RoslynDebug.Assert(value != null);`. Should we do the same here and avoid `!` suppression?

Reply:
sure.